lab_1/main.py

```diff
@@ -38,6 +85,20 @@ def calculate_frequencies(tokens: list) -> dict:
     """
     pass
 
+    if not isinstance(tokens, list):
+        return {}
+
+    freq_dict = {}
+    for token in tokens:
+        if isinstance(token, str):
+            frequency = tokens.count(token)
+            freq_dict[token] = frequency
+    return freq_dict
+
+
+freq_dict = calculate_frequencies(tokens)
```

Comment:
There should be no function calling\using in `main.py` -> use functions on the`start.py`.